chainbridge/src/lib.rs

```diff
@@ -300,6 +306,25 @@ decl_module! {
 
             Self::vote_against(who, nonce, src_id, call)
         }
+
+        /// Evaluate the state of a proposal given the current vote threshold.
+        ///
+        /// A proposal with enough votes will be either executed or cancelled, and the status
+        /// will be updated accordingly.
+        ///
+        /// # <weight>
+        /// - weight of proposed call, regardless of whether execution is performed
+        /// # </weight>
```

Comment:
is this for docs? idk how the formatting works

Reply:
Yeah, seems to be the convention in other palettes. Doesn't seem to do much in the rust docs but I think they probably parse it out for the runtime metadata, so something like a UI can use it.

Eg usage: https://crates.parity.io/pallet_democracy/struct.Module.html#method.delegate